guides/common/modules/ref_supported-clients.adoc

```diff
@@ -0,0 +1,61 @@
+[id="supported-clients_{context}"]
+= Supported Clients
+
+{Project} supports the following operating systems and architectures.
+
+Supported Host Operating Systems::
+The hosts can use the following OS versions:
+
+ifdef::orcharhino[]
+* Alma Linux
+* Amazon Linux
+endif::[]
+ifndef::satellite[]
+* CentOS
+endif::[]
+ifndef::orcharhino,satellite[]
+* CoreOS
+endif::[]
+ifndef::satellite[]
+* Debian
+ifndef::orcharhino[]
+* Fedora
+* FreeBSD
+* Junos
+* OpenSUSE
+endif::[]
+* Oracle Linux
+endif::[]
+* {EL}
+** 9.0 or later
+** 8.0 or later
+** 7.2 or later
+ifdef::satellite[]
+** 6 with the https://www.redhat.com/en/resources/els-datasheet[ELS Add-On]
+endif::[]
+ifndef::satellite[]
+* {SLES}
+ifndef::orcharhino[]
+* Solaris 8, 10
```

Comment:
Solaris 8 went EOL in 2012 according to https://computernewb.com/wiki/Oracle_Solaris_End_of_Life_Date. If you're going to list it, just list Solaris.